django_ipam/base/models.py

```diff
@@ -20,6 +27,15 @@ class AbstractIPAddress(TimeStampedEditableModel):
     subnet = models.ForeignKey(swapper.get_model_name("django_ipam", "Subnet"),
                                on_delete=models.CASCADE,
                                blank=True, null=True)
+    is_gateway = models.BooleanField(default=False)
+    used = models.BooleanField(default=False)
+    owner = models.ForeignKey(settings.AUTH_USER_MODEL,
+                              on_delete=models.CASCADE,
+                              blank=True,
+                              null=True)
+    mac_address = models.CharField(max_length=12, blank=True)
+    host = models.CharField(max_length=12, blank=True)
+    netmask = models.CharField(max_length=12, blank=True)
```

Comment:
I don't see the utility of having these two fields:

- `host` I think refers to hostname, we have this information in django-netjsonconfig as well
- `netmask`: the netmask is basically the subnet, I don't understand: why we need to have this information stored twice?